tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_run_on_start.py

```diff
@@ -46,13 +99,37 @@ def get_configuration(request):
 
 
 def test_run_on_start(get_configuration, configure_environment, restart_modulesd):
-    """
-    Check if modulesd detects the vulnerability detector scan after starting.
-
-    If we have set the parameter run_on_start to 'yes', modulesd will have to report the
-    vulnerability detector scan, and in case of the value 'no', do not report anything
-    """
-
+    '''
+    description: Check if modulesd detects the vulnerability detector scan after starting. To do this, it checks If the
+                 parameter run_on_start is set to 'yes'. Modulesd will have to report the vulnerability detector scan.
+                 In case of the value 'no', do not report anything.
+
+    wazuh_min_version: 4.2.0
+
+    parameters:
+        - get_configuration:
+            type: fixture
+            brief: Get configurations from the module.
+        - configure_environment:
+            type: fixture
+            brief: Configure a custom environment for testing.
+        - restart_modulesd:
+            type: callable
+            brief: Restart the `wazuh-modulesd` daemon.
+
+    assertions:
+        - Verify that when the `run_on_start` option is set to `yes`, the vulnerability detector module starts when
+          service starts.
+        - Verify that when the `run_on_start` option is set to `no`, the vulnerability detector module has not started.
+
+    input_description:
+        - Two use cases are found in the test module and include parameters for `run_on_start` option (`yes` and `no`).
+          The test case uses the custom_nvd_feed.json file as input file to start scanning for vulnerabilities.
+
+    expected_output:
+        - 'Could not find vulnerability starting scan log'
+        - 'Found starting scan log when run on start is disabled'
```

Comment:
```suggestion
        - 'Starting vulnerability scan'
        - 'Vulnerability scan finished'
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/54d592fab538ac1cb516607b3d9c1eea2e5d3c65)